### PR TITLE
IA Pages - remove asana integration

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
As discussed yesterday //cc @jbarrett

### CODE REVIEW

File: lib/DDGC/Web/Controller/InstantAnswer.pm [hunk 1]
```diff
@@ -911,79 +889,6 @@ sub send_to_beta :Chained('base') :PathPart('send_to_beta') :Args(0) {
     return $c->forward($c->view('JSON'));
 }
 
-sub asana :Chained('base') :PathPart('asana') :Args(0) {
```
Comment (jbarrett) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: Do we want to leave these subs in place? Or will they be moved out to a script or something at some point?

Wondering if they'll be used again in this controller.

**Comment by jbarrett** [comment]
Besides comment, :+1: 

Thanks!

File: lib/DDGC/Web/Controller/InstantAnswer.pm (hunk 1 above)
Comment (MariagraziaAlastra) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: It sounds like we don't really know what we want the asana integration behaviour to be, so the code might change as well in the future, even if it remains in the controller. Not sure

Comment (jbarrett) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: Good enough - it's in git history if we really need it.

Happy to merge?

Comment (MariagraziaAlastra) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: Yup thanks!

**Pull request merged**